profiles/package.mask

```diff
@@ -328,18 +332,20 @@ sci-mathematics/agda-lib-ffi
 >=dev-haskell/aeson-1.5
 >=dev-haskell/free-5.1.4
 >=dev-haskell/random-1.2
->=dev-haskell/splitmix-0.1
->=dev-haskell/strict-0.4
 >=dev-haskell/th-abstraction-0.4
->=dev-haskell/these-1.1
->=dev-haskell/quickcheck-instances-0.3.23
+>=dev-haskell/binary-instances-1.0.1
+>=dev-haskell/zxcvbn-hs-0.3.0.0
+>=dev-haskell/github-0.23
+>=dev-haskell/github-backup-1.20200721
+>=dev-haskell/cabal-install-parsers-0.3.0.1
```

Comment:
This masks all `cabal-install-parsers`.

```
$ emerge -pve1 @all-haskell '>=quickcheck-2.14' --ignore-built-slot-operator-deps=y

These are the packages that would be merged, in order:

Calculating dependencies... done!

!!! All ebuilds that could satisfy "dev-haskell/cabal-install-parsers" have been masked.
!!! One of the following masked packages is required to complete your request:
- dev-haskell/cabal-install-parsers-0.4::haskell (masked by: package.mask)
/gentoo-ebuilds/gentoo-haskell/profiles/package.mask:
# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2020-12-22)
# There are a large number of packages that are not ready for this upgrade yet.
# Until more of the reverse-dependencies update to the newer version, this will
# remain masked
```

I don't understand why new aeson (or quickcheck) version suddenly masks already installable package. Something is odd here.

Reply:
This was an error on my part, good catch!

The real need here is to temporarily mask `USE="test"` for `binary-instances` due to the `quickcheck` depenedency.

instead, I'll just drop this commit, a the test is already restricted and the quickcheck issue is resolved after the newer quickcheck is unmasked.